src/transformStateWithClones.js

```diff
@@ -7,7 +7,29 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const copy = { ...state };
+  const arr = [];
+
+  for (const command of actions) {
```

Comment:
use singular noun from your plural one. It will give more sense 
```suggestion
  for (const action of actions) {
```